dev/com.ibm.ws.cdi.ejb_fat/fat/src/com/ibm/ws/cdi/ejb/apps/interceptors/InterceptorsTestServlet.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cdi.ejb.apps.interceptors;
+
+import javax.inject.Inject;
+import javax.servlet.annotation.WebServlet;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+
+@WebServlet("/")
+public class InterceptorsTestServlet extends FATServlet {
+
+    @Inject
+    private RequestScopedBean bean;
+
+    @Test
+    public void testEJB() {
+        bean.message();
+    }
```

Comment:
Nothing is asserted.

I realise the the reported problem was that CDI threw an exception trying to find the interceptor, but it would be good to assert that the interceptors were called (i.e. the expected code path actually ran) or at least add a comment saying what this test is testing for.